sci-libs/libgeotiff/libgeotiff-1.7.3.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Library for reading TIFF files with embedded tags for geographic information"
+HOMEPAGE="https://trac.osgeo.org/geotiff/ https://github.com/OSGeo/libgeotiff"
+SRC_URI="https://download.osgeo.org/geotiff/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0/5"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos"
+IUSE="doc jpeg zlib" # add tiff when resolved: https://github.com/OSGeo/libgeotiff/issues/125
+
+DEPEND="
+	>=sci-libs/proj-6.0.0:=
+	>=media-libs/tiff-3.9.1:=
+	jpeg? ( media-libs/libjpeg-turbo:= )
+	zlib? ( sys-libs/zlib )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=media-libs/tiff-3.9.1
+	doc? ( app-text/doxygen )
+"
+
+src_configure() {
+	# in >1.7.3 there is BUILD_{DOC,MAN}, it should be added
+	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=ON
+		-DWITH_JPEG=$(usex jpeg)
+		-DWITH_TIFF=ON # bug 837287
+		-DWITH_TOWGS84=OFF # default
+		-DWITH_ZLIB=$(usex zlib)
+	)
+
+	cmake_src_configure
+}
+
+src_compile() {
+	cmake_src_compile
+
+	if use doc; then
+		mkdir -p docs/api || die
+		cp "${FILESDIR}"/Doxyfile Doxyfile || die
+		doxygen -u Doxyfile || die "updating doxygen config failed"
+		doxygen Doxyfile || die "docs generation failed"
+	fi
+}
+
+src_test() {
```

Comment:
Where do these steps come from?

I'll note that it looks like tests are wired up in upstream autotools but not in their CMake.

Reply:
there are build systems out there to which autotools would be a lesser evil, cmake is not one of them.

Reply:
Depends on the CMake ;)

anyway, yes, keep it as CMake but add a comment in src_test saying those are from README.

also file a bug upstream for that missing wiring

Reply:
Your bug talks about autotools, not CMake...